packages/drivers/driver-base/src/documentDeltaConnection.ts

```diff
@@ -412,14 +415,14 @@ export class DocumentDeltaConnection
 		this.disconnect(
 			createGenericNetworkError(
 				// pre-0.58 error message: clientClosingConnection
-				"Client closing delta connection",
-				{ canRetry: true },
+				error?.message ?? "Client closing delta connection",
+				{ canRetry: error === undefined },
```

Comment:
we should copy this from the provided error. If no error, then set canRetry as true.

Reply:
This might be possible if we can satisfy this comment; https://github.com/microsoft/FluidFramework/pull/24528#discussion_r2193183380